### PR TITLE
[DOC] Update docs after we upgrade xgboost4j jars to DMLC 1.6.1[skip ci]

### PR DESCRIPTION
**Pull request by nvliyuan**
since we update xgboost4j jars from [nvidia shipped version](https://repo1.maven.org/maven2/com/nvidia/xgboost4j-spark_3.0/1.4.2-0.3.0/) to [DLMC XGBoost 1.6.1](https://github.com/dmlc/xgboost), we need to update reference docs

### CODE REVIEW

File: docs/get-started/getting-started-alluxio.md [hunk 1]
```diff
@@ -221,7 +221,9 @@ NM_hostname_2
 
    - Verify Alluxio
 
+      <!-- markdown-link-check-disable -->  
       To verify that Alluxio is running, visit [http://RM_hostname:19999](http://RM_hostname:19999)
```
Comment (viadea) [inline docs/get-started/getting-started-alluxio.md:225]: maybe we should remove the link here.

**Review by viadea — CHANGES_REQUESTED** [review]

File: docs/get-started/getting-started-alluxio.md (hunk 1 above)
Comment (nvliyuan) [reply docs/get-started/getting-started-alluxio.md]: updated

**Comment by viadea** [comment]
@sameerz  @GaryShen2008  I am fine with the changes. Do you want to take a look at well?

**Comment by viadea** [comment]
@nvliyuan  could you make sure the jupyter notebook works on Dataproc using "JupyterLab"?
If so, I am fine with keeping 1 Jupyter version notebook for GCP.

**Comment by nvliyuan** [comment]
> Why are we removing the Zeppelin notebook?
> 
> Does PR only change the reference to DMLC only, or also update the mortgage example?

Because we don't have any Zeppelin notebooks in the spark-rapids-example repo, all provided demos are based on Jupiter notebook, maybe we should keep the demo in the same form. This is the only one zpln notebook in both repo, and the demo is the same.
The code also needs to update because the introduction is outofdate, the output log is different, the api need to update(Nvidia XGBoost:`setFeaturesCols`. -> DMLC: `setFeaturesCol`)

**Comment by nvliyuan** [comment]
> @nvliyuan could you make sure the jupyter notebook works on Dataproc using "JupyterLab"?
> If so, I am fine with keeping 1 Jupyter version notebook for GCP.

It indeed has some issues in some situations with the DMLC 161 jar(seems if we set spark.task.resource.gpu.amount less than 1, it will schedule more than 1 task to a executor and it will hang) , mark this pr as draft and need more time to verify

**Comment by nvliyuan** [comment]
> > @nvliyuan could you make sure the jupyter notebook works on Dataproc using "JupyterLab"?
> > If so, I am fine with keeping 1 Jupyter version notebook for GCP.
> 
> It indeed has some issues in some situations with the DMLC 161 jar(seems if we set spark.task.resource.gpu.amount less than 1, it will schedule more than 1 task to a executor and it will hang) , mark this pr as draft and need more time to verify

see https://github.com/dmlc/xgboost/issues/7986

**Review by GaryShen2008 — APPROVED** [review]

**Comment by GaryShen2008** [comment]
As I discussed with Yuan, the issue blocks the demo notebook to run successfully. So, I'd like to skip this PR in 22.06.

**Review by viadea — DISMISSED** [review]

**Comment by viadea** [comment]
@sameerz  @GaryShen2008  I approved this PR. Do you have any concern or comments ?

**Comment by viadea** [comment]
@nvliyuan  lots of old commits are shown in this PR. Do you want to try this workaround to see if it works?
https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch

**Comment by sameerz** [comment]
Please fix (or resubmit) the PR so we can review only the changes intended.

**Comment by viadea** [comment]
@nvliyuan Could you make sure the mortgage examples match the new version in https://github.com/NVIDIA/spark-rapids/pull/5990/files which @SurajAralihalli drafted?
I am thinking of merging the 2 PRs together in this PR.

**Comment by nvliyuan** [comment]
@sameerz @viadea just merged #5778 and #5990 to #6132

**Pull request closed without merging**